services/movie_session.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+    movie_show_time: datetime,
+    movie_id: int,
+    cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(
+    session_date: str | None = None,
+) -> QuerySet:
+    queryset = MovieSession.objects.all()
+
+    if session_date:
+        queryset = queryset.filter(
+            show_time__date=datetime.strptime(
+                session_date,
+                "%Y-%m-%d"
+            ).date()
+        )
+
+    return queryset
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
```

Comment:
The `get_movie_session_by_id` function uses `.get()`, which will raise a `MovieSession.DoesNotExist` exception if no session with the given ID is found. Consider handling this exception or updating the return type annotation to indicate that an exception might be raised.